packages/image_picker/image_picker_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.8.7+6
+
+* Fix NullPointerException on Android 10 devices when using Android Photo Picker
```

Comment:
Please follow the CHANGELOG style guide linked in the checklist.

Reply:
I have changed Fix to Fixes, and have added . at the end of the sentence